src/main.zig

```diff
@@ -181,6 +184,11 @@ fn parseArgs(allocator: std.mem.Allocator) !Args {
         \\    --sigma1 <f32>    Sigma 1 for DoG filter (default: 0.5)
         \\    --sigma2 <f32>    Sigma 2 for DoG filter (default: 1.0)
         \\-b, --brightness_boost <f32>   Brightness boost (default: 1.0)
+        \\    --full_characters          Uses full spectrum of characters in image.
+        \\    --ascii_chars <str>        Use what characters you want to use in the image. (default: " .:-=+*%#@")
+        \\    --sorted_ovr               Prevents sorting of the ascii_chars by size.
```

Comment:
Can you rename this to --disable_sort or something?

Reply:
Renamed.